mage/changelog.md

```diff
@@ -4,6 +4,8 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+- added node similarity algorithms (Jaccard, overlap and cosine) in C++
```

Comment:
- Similarity algorithms (jaccard, overlap and cosine) have been rewritten in C++ to improve performance.